.gitattributes

```diff
@@ -0,0 +1,11 @@
+# Set default behavior to automatically normalize line endings.
+* text=auto
+
+# Force batch scripts to always use CRLF line endings so that if a repo is accessed
+# in Windows via a file share from Linux, the scripts will work.
+*.{cmd,[cC][mM][dD]} text eol=crlf
```

Comment:
What's this magic. I trust this works and not break Linux/Mac users 🙃

Reply:
Both .cmd and .bat files are Windows command scripts. On Windows their line endings have to be crlf for being processed correctly.